### ISSUE TITLE
SOME TESTS FAILURES

### ISSUE THREAD

**Issue by craice** [ITDP/the-online-brt-planning-guide#15]
```
results: SOME TESTS FAILURES (success: false)
tests.Test_01_Tools
  test_003_line_positions: ERROR E
    [file_open,/tmp//a]
Called from C:\HaxeToolkit\haxe\std/neko/_std/sys/io/File.hx line 54
Called from C:\HaxeToolkit\haxe\std/neko/_std/sys/io/File.hx line 38
Called from tests/Test_01_Tools.hx line 78
Called from C:\HaxeToolkit\haxe\std/neko/_std/Reflect.hx line 58
tests.Test_03_Parser
  test_021_include: ERROR E
    [file_open,/tmp//b]
Called from C:\HaxeToolkit\haxe\std/neko/_std/sys/io/File.hx line 54
Called from C:\HaxeToolkit\haxe\std/neko/_std/sys/io/File.hx line 38
Called from tests/Test_03_Parser.hx line 584
Called from C:\HaxeToolkit\haxe\std/neko/_std/Reflect.hx line 58
```

**Comment by jonasmalacofilho** [comment]
This happens on Windows, with a recent `development` checkout.

At the moment, these unit test failures are harmless, but it does mean we're not fully running those tests on Windows.

**Comment by jonasmalacofilho** [comment]
Fixed in 8116f5960e7fdebcd4a5cbad648e98763f8eb10f and e44d45f9c70d29ce3ebac4f73b3d4f369bdbfb47
